Makefile

```diff
@@ -30,8 +30,13 @@ format:
 	black -l 100 $(python-packages)
 	isort $(python-packages)
 
-test:
-	python3 -m pytest -n$(shell nproc) -vv tests
+test: test_unit test_integration
+
+test_unit:
+	python3 -m pytest -vv tests
+
```

Comment:
Why is the parallelization removed?

Reply:
If GitHub PRs would show commit messages, you'd know :wink: . [It makes not sense with the few tests we have.](https://github.com/Componolit/python-style/pull/14/commits/ee2513114b128b59fd0fe464d506a8075a1df75f).

Reply:
I see.